strings/names/last.txt

```diff
@@ -1,851 +1,1063 @@
-	Rodriguez
-	Lopez
-	Gomez
-	Driscoll
-	Foley
+	Mayberry
 	McNab
-	O'Malley
-	Lynch
-	Moloney
-	Bologna
-	Scully
-	Wagner
-	Bauer
-	Meier
-	Schneider
-	Jaeger
+	Mccloud
+	Mcclymonds
+	Mccullough
+	Mccune
+	Mcdonald
+	Mcfall
+	Mcintosh
+	Mckendrick
+	Mcloskey
+	Mcmullen
+	Medicine Crow
+	Meier
+	Melnyk
+	Mendis
+	Mendoza
+	Meng
+	Mens
+	Merryman
+	Mertens
+	Metzer
 	Metzger
-	Gaumond
-	Walmund
-	Hubert
-	Hanlon
-	Agnor
-	Dardar
-	Alvin
-	Samsa
-	Gruber
+	Meyers
+	Mifsud
+	Mikhail
+	Mikhaylov
+	Miki
+	Mildred
+	Miller
+	Millhouse
+	Mills
+	Milne
+	Ming
+	Mingle
+	Minnie
+	Mirza
+	Mitchell
+	Moberly
+	Modi
+	Moghadam
+	Moloney
+	Monti
+	Montoya
+	Moon
+	Moore
+	Morcos
+	Morgan
+	Morris
+	Mortland
+	Morton
+	Mosser
+	Mueller
+	Muggins
+	Mugheri
+	Mull
+	Muller
+	Murakami
+	Murdock
+	Murphy
+	Murray
+	Nader
+	Nadir
+	Nahas
+	Naidu
+	Naifeh
+	Naik
+	Nair
+	Najjar
+	Nakamura
+	Nanda
+	Naquin
+	Narain
+	Narang
+	Narasimhan
+	Nasir
+	Naser
+	Nash
+	Nassar
+	Nazari
+	Neely
+	Nehling
+	Newbern
+	Newton
+	Ngo
+	Nguyen
+	Nicholas
+	Nickolson
+	Nielsen
+	Nishimura
+	Northey
+	Noton
 	Novik
-	Jafarov
-	Dubois
-	Dupont
-	Lambert
-	Mertens
+	Nowak
+	Nunez
+	O'Conner
+	O'Malley
+	Obi
+	Ogawa
+	Okada
+	Okamoto
+	Okoye
+	Old Fire Heart
+	Olphert
+	Olsen
+	Oneal
+	Oppenheimer
+	Ortiz
+	Osman
+	Osteen
+	Osterweis
+	Osterwise
+	Ota
+	Otis
+	Overstreet
+	Owen
+	Owens
+	Ozdemir
+	Padmanabhan
+	Page
+	Pai
+	Pain
+	Pallas
+	Palmer
+	Park
+	Parker
+	Parkinson
+	Parlsey
+	Patel
+	Patterson
+	Paul
+	Pavlov
+	Paynter
+	Pearsall
 	Peeters
-	Kovacevic
-	Vujic
+	Peng
+	Pennington
+	Perkins
+	Pershing
+	Peters
+	Petrov
 	Petrovic
+	Petrus
+	Pfeifer
+	Philips
+	Phillips
+	Picard
+	Pinney
+	Pinto
+	Plankchest
+	Poehl
+	Poirot
+	Poley
+	Polson
+	Potter
+	Powell
+	Power
+	Powers
+	Prabhu
+	Pradhan
+	Prakash
+	Pratt
+	Pratt
+	Prechtl
+	Prescott
+	Prevatt
+	Price
+	Priebe
+	Prieto
+	Pritchard
+	Puente
+	Punchbeef
+	Pycroft
+	Qin
+	Qiu
+	Quinn
+	Quirin
+	Quraishi
+	Qureshi
+	Radium
+	Rader
+	Rahal
+	Rahl
+	Ramos
+	Randolph
+	Ratcliff
+	Raub
+	Ray
+	Reade
+	Reddy
+	Reichard
+	Reid
+	Reighner
+	Reza
+	Rhinehart
+	Rhodes
+	Richards
+	Richardson
+	Richter
+	Rifler
+	Riggle
+	Riker
+	Ring
+	Rizzi
+	Roach
+	Roadman
+	Roberts
+	Robertson
+	Robinson
+	Roby
+	Rockwell
+	Rodriguez
+	Rogers
+	Rohtin
+	Romeo
+	Rose
+	Rosensteel
+	Rossi
+	Rowley
+	Roy
+	Russell
+	Ryals
+	Sabbag
+	Sabbagh
+	Safar
+	Sagan
+	Sahin
+	Said
+	Saito
+	Sakamoto
+	Sakara
+	Salami
+	Saleeby
+	Salib
+	Saliba
+	Salman
+	Samaha
+	Samsa
+	Sanders
+	Sandford
+	Sandys
+	Sarraf
+	Sasaki
+	Sato
+	Sauter
+	Sayegh
+	Saylor
+	Schaeffer
+	Schere
+	Schmidt
+	Schmuck
+	Schneider
+	Schofield
+	Schrader
+	Scott
+	Scully
+	Sealis
+	Seelig
+	Seidner
+	Seif
+	Sellers
+	Semenov
+	Seth
+	Shadid
+	Shafer
+	Shaffer
+	Shah
+	Shalhoub
+	Shammas
+	Shamon
+	Shamoun
+	Shaner
+	Shao
+	Shaw
+	Sheets
+	Shen
+	Shibata
+	Shick
+	Shimizu
+	Shirey
+	Sholl
+	Shupe
+	Sideiron
+	Sidower
+	Siegrist
+	Silver
+	Simmons
+	Simpson
+	Simsek
+	Singh
+	Sitting Bull
```

Comment:
It seems in poor taste to just casually add a couple names from some well-known Native Americans, just for the sake of diversifying our name pool. Especially as (1) some of these names were given based on the specific individual (e.g. Sitting Bull) (2) some of these names seem to be questionable translations (e.g. Crow Dog). It just sits weird with me. Would appreciate thoughts from those who are Native American/Indigenous.

Reply:
Names drawn from [https://www.marquette.edu/library/archives/Mss/BCIM/BCIM-SC1-NAsurnames.pdf], but also would appreciate this if possible.

Reply:
Thanks!

Reply:
> Thanks!

If you'd like, I can remove them and wait until we get feedback to see if it should be added? Only about... 5 or so names to remove, so it wouldn't be much to remove if this is the way you'd prefer!

Reply:
Yeah, it just feels weird to me for us / the game to be giving random players these names. That sounds good!

Reply:
> Yeah, it just feels weird to me for us / the game to be giving random players these names without context / history. That sounds good!

Sure thing! I understand completely